type.js

```diff
@@ -0,0 +1,45 @@
+const writeChar = (element, char) => {
+    element.dispatchEvent(
+        new KeyboardEvent("keydown", {
+            key: char.charCodeAt(0),
+            char,
+            shiftKey: true,
+            bubbles: true
+        })
+    );
+    element.dispatchEvent(
+        new KeyboardEvent("keypress", {
+            key: char.charCodeAt(0),
+            char,
+            shiftKey: true,
+            bubbles: true
+        })
+    );
+
+    document.execCommand("insertText", true, char);
+
+    element.dispatchEvent(
+        new KeyboardEvent("keyup", {
+            key: char.charCodeAt(0),
+            char,
+            shiftKey: true,
+            bubbles: true
+        })
+    );
+};
+
+const writeOnElement = (element, state) => {
+    // Assuming words are 5 char in average
+    const cpm = state.wpm * 5;
+    const timeToWrite = 60 / cpm;
+
+    let index = 0;
+    interval = setInterval(() => {
+        writeChar(element, lorem.text[index]);
```

Comment:
Not required but I don't like this that much, feels like magic. I would rather have it in the same file.

Reply:
Looks like this is the way they [designed it to work](https://stackoverflow.com/a/7568862/2161256), We can probably set up webpack or something like that but ain't nobody got time for that.

Reply:
Or just use one file 😅